README.md

```diff
@@ -22,44 +23,106 @@ The main purpose of the library is to handle 4 key functions:
 4. Maintaining and providing logs from a jobs output.
 
 ``` go
+type JobStatus struct {
+	// Options - ["Stopped", "Completed", "Errored"]
+	status string
+	exitCode uint8
+	output JobOutput
+}
+
+type JobOutput struct {
+	StdOut []byte
+	StdErr []byte
+}
+
+type JobsInfo struct {
+	Command *exec.Cmd
+	JobStatus JobStatus
+}
+
+// Note: All channels will be buffered with cap 1.
+type JobChans struct {
+	// Write from API
+	Kill chan struct{}{}
+	// Read from API
+	Status chan JobStatus
+}
+
 type JobsManager struct {
-	Jobs sync.Map
-
-	// These fields are used for pseduo-persistence.
-	// Type: uuid.UUID -> []byte
-	StdOut sync.Map
-	StdErr sync.Map 
-
-	// We store status locally because PIDs can cycle.
-	// This could cause integerity concerns.
-	// e.g Job A with PID 1 completes but Job B cycles back to PID 1
-	Status sync.Map
-	ExitCode sync.Map
+	JobInfo sync.Map // uuid -> JobsInfo
+	JobChannels sync.Map
 }
 ```
 
 ## Start
 ```go
-func (*JobsManager) Start(string, ...string) (uuid.UUID, bool)
+func (*JobsManager) Start(cmd string, args ...string) (uuid.UUID)
```

Comment:
Cannot this call return an error?

Reply:
**Potentially based on input validation of the parameters, but otherwise I don't think so.**
- The reason I removed it was because the go-routine worker that this function would spin up would be the one to report errors.
- This function only has the job of creating the configuring the command structure and passing it to the Worker go-routine.